### PR TITLE
Fix plugin channel remap 1.13 -> 1.12 🙌

### PR DESCRIPTION
**Pull request by ForceUpdate1**
Solution for the issue #117

### CODE REVIEW

**Comment by ColaIan** [comment]
Please merge it, it fixes bungeecord messaging channel.

File: core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_12_2to1_13/packets/PlayerPacket1_13.java [hunk 1]
```diff
@@ -87,6 +90,20 @@ public void handle(PacketWrapper wrapper) throws Exception {
...
+
+                            if (oldChannel.equals("REGISTER") || oldChannel.equals("UNREGISTER")) {
+                                String[] channels = new String(wrapper.read(Type.REMAINING_BYTES), StandardCharsets.UTF_8).split("\0");
+                                List<String> rewrittenChannels = new ArrayList<>();
+                                for (int i = 0; i < channels.length; i++) {
+                                    String rewritten = InventoryPackets.getOldPluginChannelId(channels[i]);
+                                    if (rewritten != null) {
+                                        rewrittenChannels.add(rewritten);
+                                    } else if (!Via.getConfig().isSuppress1_13ConversionErrors() || Via.getManager().isDebug()) {
+                                        Via.getPlatform().getLogger().warning("Ignoring plugin channel in incoming REGISTER: " + channels[i]);
```
Comment (creeper123123321) [inline core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_12_2to1_13/packets/PlayerPacket1_13.java]: Should use viabackwards logger and this is an outgoing message

File: core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_12_2to1_13/packets/PlayerPacket1_13.java [hunk 2]
```diff
@@ -433,8 +450,25 @@ public void handle(PacketWrapper wrapper) throws Exception {
...
+
+                            if (newChannel.equals("minecraft:register") || newChannel.equals("minecraft:unregister")) {
+                                String[] channels = new String(wrapper.read(Type.REMAINING_BYTES), StandardCharsets.UTF_8).split("\0");
+                                List<String> rewrittenChannels = new ArrayList<>();
+                                for (int i = 0; i < channels.length; i++) {
+                                    String rewritten = InventoryPackets.getNewPluginChannelId(channels[i]);
+                                    if (rewritten != null) {
+                                        rewrittenChannels.add(rewritten);
+                                    } else if (!Via.getConfig().isSuppress1_13ConversionErrors() || Via.getManager().isDebug()) {
+                                        Via.getPlatform().getLogger().warning("Ignoring plugin channel in outgoing REGISTER: " + channels[i]);
```
Comment (creeper123123321) [inline core/src/main/java/nl/matsv/viabackwards/protocol/protocol1_12_2to1_13/packets/PlayerPacket1_13.java]: Should use viabackwards logger and this is an incoming message

**Comment by MylesIsCool** [comment]
Ty

**Pull request merged**